sql/values_test.go

```diff
@@ -18,74 +18,60 @@
 package sql
 
 import (
+	"fmt"
 	"reflect"
-	"strconv"
 	"testing"
 
-	"github.com/cockroachdb/cockroach/sql/driver"
 	"github.com/cockroachdb/cockroach/sql/parser"
 )
 
-func TestProcessSelect(t *testing.T) {
-	s := (*Server)(nil) // enough for now
+func TestValues(t *testing.T) {
+	p := &planner{}
```

Comment:
Looks like none of the `planner` methods need to take a pointer; should they be changed to have a value receiver?

I'm surprised having a pointer receiver is enough to cause the object to be heap allocated even when none of those methods even access it.